reference/src/main/java/com/google/android/fhir/reference/PatientDetailsFragment.kt

```diff
@@ -74,32 +79,117 @@ class PatientDetailsFragment : Fragment() {
         .get(PatientDetailsViewModel::class.java)
     patientDetailsViewModel.livePatientData.observe(viewLifecycleOwner) { setupPatientData(it) }
     patientDetailsViewModel.livePatientObservation.observe(viewLifecycleOwner) {
-      adapter.submitList(it)
+      val observations = it.map { observation -> Model(observation.code, observation.value) }
+      renderCard(resources.getText(R.string.header_observation), observations)
+    }
+
+    patientDetailsViewModel.livePatientCondition.observe(viewLifecycleOwner) {
+      val observations = it.map { observation -> Model(observation.code, observation.value) }
+      renderCard(resources.getText(R.string.header_conditions), observations)
     }
-    binding.apply { addScreener.setOnClickListener { onAddScreenerClick() } }
   }
 
+  private fun renderCard(header: CharSequence, observations: List<Model>) {
```

Comment:
```suggestion
  private fun renderCard(header: CharSequence, models: List<Model>) {
```

this is not limited to observation so use generic names for variables.

Reply:
done